src/components/UserActions/ClickedDocumentList.ts

```diff
@@ -96,6 +96,11 @@ export class ClickedDocumentList extends Component {
 
         this.options = ComponentOptions.initComponentOptions(element, ClickedDocumentList, options);
 
+        if (!this.options.userId) {
```

Comment:
Check also for `UserActivity` and `QueryList` component. They might suffer the same issue.